### PR TITLE
[Phone] TeamsActivityHandler support

### PR DESCRIPTION
**Pull request by bobokids**
### Purpose
*What is the context of this pull request? Why is it being done?*
Support TeamsActivityHandler for Event and Phone skill. 
Update the Cancel/Logout handling

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs [hunk 1]
```diff
@@ -39,5 +40,40 @@ await GetTestFlow()
...
+                Type = ActivityTypes.Event,
+                Name = "OutgoingCall",
+            };
+
+            await GetSkillTestFlow()
+               .Send(activity)
+               .AssertReply(Message(OutgoingCallResponses.RecipientPrompt))
+               .Send(GeneralUtterances.Cancel)
+               .AssertReply(Message(PhoneMainResponses.CancelMessage))
+               .StartTestAsync();
```
Comment (xieofxie) [inline skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs]: check ActionResult here? Same for Test_Action_Help_Interruption

Comment (bobokids) [reply skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs]: fixed.

File: skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs [hunk 2]
```diff
@@ -39,5 +43,51 @@ await GetTestFlow()
...
+               .AssertReply(Message(OutgoingCallResponses.ExecuteCall, new Dictionary<string, object>()
+               {
+                   { "contactOrPhoneNumber", "Bob Botter" },
+               }))
+               .AssertReply(OutgoingCallEvent(new OutgoingCall
+               {
+                   Number = "555 666 6666",
+                   Contact = StubContactProvider.BobBotter,
+               }))
+               .StartTestAsync();
```
Comment (xieofxie) [inline skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs]: miss a SkillActionEndMessage here

Comment (bobokids) [reply skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs]: fixed.

**Pull request merged**